.github/workflows/run_unit_tests_and_review_pr.yml

```diff
@@ -0,0 +1,68 @@
+name: Run unit tests and review pull request
```

Comment:
Is it for the shared module only?

Reply:
@markgravity I'm not sure I understand your question correctly but we cannot run `./gradlew detekt` and `./gradlew lint` inside of `android` package. And if we run this command from the root of project, I believe it will run all kotlin files in the project. Please correct me if I'm wrong. Or is there another approach we can run inside of the `android` package?  🙇🏻‍♀️  cc @luongvo

Reply:
Both Android and shared modules fyi @markgravity @lydiasama

Some tasks can not be separated between modules.